xml/gnome_custom.xml

```diff
@@ -133,17 +121,16 @@
    <itemizedlist>
     <listitem>
      <para>
-      <guimenu>Language</guimenu>. Choose a language from the list and confirm
-      with <guimenu>Done</guimenu>. To make the change take effect, you need to
-      restart the desktop afterwards.
+      <guimenu>Language</guimenu>. Choose a language from the list. To make the
+      change take effect, you will be prompted to restart the desktop session
```

Comment:
I suggest that the wording at line 267:

"you must log out of your session and log back in to activate the changes."

... is clearer. So we could say:

"You will be prompted to [literal]Restart the session for changes to take effect[/literal]. You must log out of your session and log back in (or reboot) to do so."

The (or reboot) is optional. It is possibly obvious enough not to mention.